website/static/js/registrationUtils.js

```diff
@@ -306,18 +311,17 @@ var Draft = function(params, metaSchema) {
 
     self.urls = params.urls || {};
 
-    // TODO: uncomment to support draft approval states
-    //    self.fulfills = params.fulfills || [];
-    //    self.isPendingReview = params.flags.isPendingReview || false;
-    //    self.requiresApproval = params.config.requiresApproval || false;
-    //    self.isApproved = params.flags.isApproved || true;
-    //
-    //   $.each(params.config || {}, function(key, value) {
-    //        self[key] = value;
-    //    });
-    //    $.each(params.flags || {}, function(key, value) {
-    //        self[key] = value;
-    //    });
+
+    self.fulfills = params.fulfills || [];
+    self.isPendingReview = params.is_pending_review;
+    self.isApproved = params.is_approved;
+
+    self.requiresApproval = ko.pureComputed(function() {
+        return self.metaSchema && self.metaSchema.requiresApproval;
```

Comment:
This does not appear to depend on any observables, and as such may not need to be a pureComputed.